x-pack/plugins/uptime/common/types/index.ts

```diff
@@ -44,3 +44,19 @@ export type SyntheticsMonitorSavedObject = SimpleSavedObject<{
     };
   };
 }>;
+
+interface LocationGeo {
+  lat: number;
+  lon: number;
+}
+
+export interface ManifestLocation {
+  url: string;
+  geo: {
+    name: string;
+    location: LocationGeo;
+  };
+  status: string;
```

Comment:
What is the status for, for my information?

Reply:
i think if the location is in GA or beta etc.